### PR TITLE
Improve aside feed bar

### PR DESCRIPTION
**Pull request by math-GH**
The feed sidebar needs some improvements under the hood in HTML

where this improvement started....

## hover background = clickable link
before:
(see the mouse pointer. hand = hyperlink, arrow = no hyperlink)
link-area-before

after:

link-area-after

## Expand category
Before:
bad used `<a href="#">`  for the expanding icon. It is not a real link.
grafik

After:
`<button>` instead of `<a>`. This button has a much more bigger clickable area. It has got a `title="expand category"`. The icon of this button changes color shade, while hovering over the button but does not change while hovering over the category name
grafik

## mute icon

before: 
Mute icons have impact on the aligned feeds:
grafik

after:
feeds are aligned. Mute icon is placed still good visible between favicon and feed name
grafik

## mouse over text feed
before:
If the feed is empty (orange) then a mouse over text is shown.
No information is given what the red icon means
grafik

If the feed has an error, an information text is displayed
grafik

after:
mouse over text:
"open feed [+feed name]" (important for long feed names, that are cropped at the end. So the complete feed name is readable. This text highlights that the user can open here the feed to read it.
Information is given, if the feed ist empty
Information is given, if the feed is muted
Information ist given, if there is an error
grafik

Error information is given. The "triangle warning" icon is added
grafik

## feed drop down menu

before:
if a feed is empty a mouse over text is shown whole hovering over the drop down menu
grafik

after:
no mouse over text
grafik

## mouse over between feed name/link and cog icon/drop down menu icon
after:
the feed drop down icon has another mouse over text than the feed name/link.
The cog icon has a bit transparency while hovering over the name and has no transparency while hovering over the drop down menu icon. It is very useful for the user to understand that there are 2 different actions near in 1 line.
mouseover-feed-and-menu

similar on category level

mouseover-category

## favicon is linked too
before:
the favion was no link to open the feed

after:
it is now linked too

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

File: app/i18n/fr/gen.php [hunk 1]
```diff
@@ -29,6 +29,9 @@
 		'load_default_shortcuts' => 'Utiliser les raccourcis par défaut',
 		'manage' => 'Gérer',
 		'mark_read' => 'Marquer comme lu',
+		'menu' => array(
+			'open' => 'Open menu',	// TODO
```
Comment (Alkarex) [inline app/i18n/fr/gen.php]:
```suggestion
			'open' => 'Ouvrir le menu',
```

File: app/i18n/fr/sub.php [hunk 2]
```diff
@@ -29,7 +29,9 @@
 			'help' => 'Fournir l’URL d’un <a href="http://opml.org/" target="_blank">fichier OPML</a> qui donnera dynamiquement la liste des flux de cette catégorie',
 		),
 		'empty' => 'Catégorie vide',
+		'expand' => 'Expand category',	// TODO
```
Comment (Alkarex) [inline app/i18n/fr/sub.php]:
```suggestion
		'expand' => 'Développer la catégorie',
```

File: app/i18n/fr/sub.php [hunk 3]
```diff
@@ -29,7 +29,9 @@
...
 		'information' => 'Informations',
+		'open' => 'Open category',	// TODO
```
Comment (Alkarex) [inline app/i18n/fr/sub.php]:
```suggestion
		'open' => 'Ouvrir la catégorie',
```

File: app/i18n/fr/sub.php [hunk 4]
```diff
@@ -183,9 +185,13 @@
 		'method_help' => 'Les données POST supportent automatiquement <code>application/x-www-form-urlencoded</code> et <code>application/json</code>',
 		'method_postparams' => 'Données pour POST',
 		'moved_category_deleted' => 'Lors de la suppression d’une catégorie, ses flux seront automatiquement classés dans <em>%s</em>.',
-		'mute' => 'désactivé',
+		'mute' => array(
+			'_' => 'désactivé',
+			'state_is_muted' => 'This feed is muted',	// TODO
```
Comment (Alkarex) [inline app/i18n/fr/sub.php]:
```suggestion
			'state_is_muted' => 'Ce flux est désactivé',
```

File: app/i18n/fr/sub.php [hunk 5]
```diff
@@ -183,9 +185,13 @@
...
+		),
 		'no_selected' => 'Aucun flux sélectionné.',
 		'number_entries' => '%d articles',	// IGNORE
+		'open_feed' => 'Open feed %s',	// TODO
```
Comment (Alkarex) [inline app/i18n/fr/sub.php]:
```suggestion
		'open_feed' => 'Ouvrir le flux %s',
```

File: app/i18n/en/sub.php [hunk 6]
```diff
@@ -183,9 +185,13 @@
...
 		'method_postparams' => 'Payload for POST',
 		'moved_category_deleted' => 'When you delete a category, its feeds are automatically classified under <em>%s</em>.',
-		'mute' => 'mute',
+		'mute' => array(
+			'_' => 'mute',
+			'state_is_muted' => 'This feed is muted',
+		),
 		'no_selected' => 'No feed selected.',
 		'number_entries' => '%d articles',
+		'open_feed' => 'Open feed %s',
```
Comment (Alkarex) [inline app/i18n/en/sub.php:194]: I would have preferred `Open feed` instead of `Open feed %s` but no big deal

Comment (math-GH) [reply app/i18n/en/sub.php:194]: I understand.
Why I chose with the `%s`: It is the tooltip that shows the complete feed name, that could be cropped in the side bar.

**Comment by Alkarex** [comment]
@math-GH Ups, it looks like the menu for the labels is not working anymore

image

File: app/i18n/en/sub.php (hunk 6 above)
Comment (Alkarex) [reply app/i18n/en/sub.php:194]: Alright, makes sense. Would be good to have the same for the user labels

**Comment by math-GH** [comment]
@Alkarex Thanks! It led me to another potential bug :)

**Comment by Alkarex** [comment]
Regression https://github.com/FreshRSS/FreshRSS/issues/6735

**Comment by Alkarex** [comment]
Another regression https://github.com/FreshRSS/FreshRSS/issues/6737

**Pull request merged**